docs/proposals/202105256-cluster-class-and-managed-topologies.md

```diff
@@ -233,23 +242,31 @@ type ClusterClassSpec struct {
   // Note: Patches will be applied in the order of the array.
   // +optional
   Patches []ClusterClassPatch `json:"patches,omitempty"`
+  
+  // MachineHealthCheckClass defines a MachineHealthCheck for each 
+  // ControlPlaneClass and MachineDeploymentClass in this ClusterClass.
+  // +optional
```

Comment:
```suggestion
  // MachineHealthCheck defines the default MachineHealthCheck configuration 
  // used when creating a ControlPlane and for each MachineDeployment in this ClusterClass.
  // +optional
```

Reply:
This isn't quite right - there should only be a minimal difference between the defaulting in ClusterClass MHCs vs the defaulting in the webhook see #5871.

Reply:
By defaulting I meant that these are the parameters used when creating the MHC object

Reply:
This structure is out of sync with the proposal. I've removed it - MHCs should only be defined under the ControlPlane or under the MachineDeployments.